src/utils/content.ts

```diff
@@ -2,12 +2,11 @@ import * as fs from 'fs';
 import path from 'path';
 import glob from 'glob';
 import frontmatter from 'front-matter';
-import sbConfig from './../../stackbit.config';
 import * as types from 'types';
 
-if (!sbConfig.pagesDir || !sbConfig.dataDir) throw new Error('Invalid Stackbit config file');
-
-export const siteConfigFile = sbConfig.dataDir + '/config.json';
+export const dataDir = 'content/data';
```

Comment:
Now that Git CSI module doesn't have a distinction between contentDir & pagesDir, we'll need to change the code in this module & similar starters (but - this is outside the scope of this PR)

Reply:
got it